modules/cudacodec/src/NvEncoder.cpp

```diff
@@ -431,7 +431,7 @@ bool NvEncoder::Reconfigure(const NV_ENC_RECONFIGURE_PARAMS* pReconfigureParams)
 
 NV_ENC_REGISTERED_PTR NvEncoder::RegisterResource(void* pBuffer, NV_ENC_INPUT_RESOURCE_TYPE eResourceType,
     int width, int height, int pitch, NV_ENC_BUFFER_FORMAT bufferFormat, NV_ENC_BUFFER_USAGE bufferUsage,
-    NV_ENC_FENCE_POINT_D3D12* pInputFencePoint, NV_ENC_FENCE_POINT_D3D12* pOutputFencePoint)
+    NV_ENC_FENCE_POINT_D3D12* pInputFencePoint)
```

Comment:
In general we should have `#if` guard with version check here.

Reply:
Luckily we only call `RegisterResource` internally from `RegisterInputResources`.  `pOutputFencePoint` has always been a redundant function argument included because I mostly copied the implementation of NvEncoder from the Nvidia samples and missed the redundancy.